gov2/iam/DetachUserPolicy/DetachUserPolicyv2_test.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+    "context"
+    "encoding/json"
+    "errors"
+    "fmt"
+    "io/ioutil"
+    "testing"
+    "time"
+
+    "github.com/aws/aws-sdk-go-v2/service/iam"
+)
+
+type IAMDetachRolePolicyImpl struct{}
+
+func (dt IAMDetachRolePolicyImpl) DetachRolePolicy(ctx context.Context,
+    params *iam.DetachRolePolicyInput,
+    optFns ...func(*iam.Options)) (*iam.DetachRolePolicyOutput, error) {
+
+    output := &iam.DetachRolePolicyOutput{}
+
+    return output, nil
+}
+
+type Config struct {
+    RoleName string `json:"RoleName"`
+}
+
+var configFileName = "config.json"
+
+var globalConfig Config
+
+func populateConfiguration() error {
+    content, err := ioutil.ReadFile(configFileName)
+    if err != nil {
+        return err
+    }
+
+    text := string(content)
+
+    err = json.Unmarshal([]byte(text), &globalConfig)
+    if err != nil {
+        return err
+    }
+
+    if globalConfig.RoleName == "" {
+        msg := "You musts supply a value for RoleName in " + configFileName
+        return errors.New(msg)
+    }
+
+    return nil
+}
+
+func TestDetachRolePolicy(t *testing.T) {
+    thisTime := time.Now()
+    nowString := thisTime.Format("2006-01-02 15:04:05 Monday")
+    t.Log("Starting unit test at " + nowString)
+
+    err := populateConfiguration()
+    if err != nil {
+        t.Fatal(err)
+    }
+
+    api := &IAMDetachRolePolicyImpl{}
+
+    policyArn := "arn:aws:iam::aws:policy/AmazonDynamoDBFullAccess"
+    input := &iam.DetachRolePolicyInput{
+        PolicyArn: &policyArn,
+        RoleName:  &globalConfig.RoleName,
+    }
+
+    _, err = DetachDynamoFullPolicy(context.Background(), api, input)
+    if err != nil {
```

Comment:
"detach...to" -> "detach...from" -- throughout this file?

Reply:
Fixed